src/SASViyaApiClient.ts

```diff
@@ -949,6 +957,7 @@ export class SASViyaApiClient {
    * @param accessToken - an optional access token for an authorized user.
    * @param waitForResult - a boolean indicating if the function should wait for a result.
    * @param expectWebout - a boolean indicating whether to expect a _webout response.
+   * @param pollOptions - an object that represents poll interval and maximum amount of attempts.
```

Comment:
Could we show an example of this object here in the description?

Reply:
done